tools/samtools.wdl

```diff
@@ -637,3 +652,61 @@ task collate_to_fastq {
         maxRetries: max_retries
     }
 }
+
+task fixmate {
+    meta {
+        description: "This WDL task runs Samtools fixmate on the input BAM file. This fills in mate coordinates and insert size fields."
+    }
+
+    parameter_meta {
+        bam: "Input BAM format file to add mate information. Must be name-sorted or name-collated."
+        prefix: "Prefix for the output file. The extension will be added."
+        extension: "File format extension to use for output file. Allowable types: [.sam, .bam, .cram]"
+        add_cigar: "Add template cigar ct tag"
+        add_mate_score: "Add mate score tags. These are used by markdup to select the best reads to keep."
+        disable_proper_pair_check: "Disable FR proper pair check"
```

Comment:
IDK what this means. Can we elaborate or add an `external_help` link?

Reply:
I'll have to look it up. I also am not sure what it means. I assume that it requires there to be one forward mapped read and one reverse mapped read to make a proper pair, but that isn't clear from the help text from `samtools`.